jms/1.7/modules/ROOT/pages/jms-topic-subscription.adoc

```diff
@@ -22,13 +34,25 @@ Once a durable subscription registered, each time a consumer is available it wil
 </jms:consume>
 ----
 
-Then the JMS provider stores the messages sent or published to the topic, as it would store messages sent to a queue. If this or another application creates a 'topic-consumer' using the same connection and its client ID, the same topic, and the same subscription name, the subscription is reactivated, and the JMS provider delivers the messages that were published while the subscriber was inactive.
 
-== Allowing Multiple Consumers on the Same Subscription
 
-In JMS 1.1, a subscription on a topic was not permitted to have more than one consumer at a time, meaning that the work of processing messages on a topic subscription could not be shared, thereby limiting the scalability of the application. This restriction has been removed in JMS 2.0 by the introduction of *shared subscriptions*.
+== Configure Shared Subscriptions
+
+In JMS spec 1.1 connection, a subscription on a topic can not have more than one consumer at a time, meaning that processing work of messages on a topic subscription can not be shared, thereby limiting the scalability of the application.
+JMS spec 2.0 connection removes this restriction and enables you to configure shared consumers on the same subscription.
+
+In the following example, you configure the *Shared* field for the *Topic consumer* configuration. With this configuration on two different *On New Messages* source instances, when a message is published to the invoices destination it is delivered only to one of the sources and not the other:
```

Comment:
This wording is used earlier and I am not certain that it is precise. I think we are overusing "configure" and "configuration," to the extent that it impairs meaning. Should this really be...:
```suggestion
In the following example, you set *Topic consumer* to *Shared*.
```
The rest of this intro is confusing, because it refers to "two different *On New Messages* " instances that are not shown next, and it describes a NOT sharing situation.